packages/service/src/service.ts

```diff
@@ -27,7 +27,7 @@ const auth = (req, resp, next) => {
 
 passport.use(
   new LocalStrategy(async function (username, _password, done) {
-    const user = await createUser({ name: username });
+    const user = await createUser({ name: username, password: _password });
     return done(null, { username: user.name, id: user.id });
```

Comment:
This runs when an existing user is trying to log in. We probably want to use `validateUser` not `createUser` here. Also, we probably want to return the full user record we get back from the database (or `false` if no such user exists).